### PR TITLE
[STAKAN-8] hide useless fields in zendesk form

### PR DESCRIPTION
**Pull request by Kud8**
https://goerli.stkr.io/ - In "Help button there are updated fields"

### CODE REVIEW

**Comment by AndrewKhizhnyak** [comment]
Can you plz add a PR description and link to the Jira task just to understand what is going on here

File: src/components/ZendeskMounter/index.tsx [hunk 1]
```diff
@@ -86,6 +103,7 @@ const ZendeskMounterComponent = ({ classes = {} }: StyledComponentProps) => {
   };
   const handleShow = () => {
     ZendeskAPI('webWidget', 'show');
+    setTimeout(hideUselessFields, 500);
```
Comment (AndrewKhizhnyak) [inline src/components/ZendeskMounter/index.tsx:106]: why a 0.5 sec delay is here?

Comment (Kud8) [reply src/components/ZendeskMounter/index.tsx:106]: wait render of zendesk iframe content. We can't get any ref of callback on it, therefore I put some timeout

**Comment by Kud8** [comment]
https://goerli.stkr.io/ - In "Help button there are updated fields"

**Review by AndrewKhizhnyak — APPROVED** [review]
Approve but wanna see PR description

**Pull request merged**